analyzers/ResonanceBuilder.py

```diff
@@ -6,7 +6,7 @@
 import pprint 
 import itertools
 
-mass = {23: 91, 24: 80.4, 25: 125}
+mass = {23: 91, 25: 125, 32: 1000}
```

Comment:
you should not remove 24 (the W). 
What is 32?

Reply:
32 is Z'